### DISCUSSION TITLE
Strapi as a database

### DISCUSSION THREAD

**Discussion by VirAlex** [strapi/strapi#7842] [Legacy Topics]
I have a quick question about Strapi. I use it quite a bit with Gatsby or Vue in CMS mode. But my question is, is it possible to store data from the front on Strapi (eg pseudo name, ...) in database mode. And thus be able to recover all that by the interface of Strapi? Thank you for your advice.

**Comment by derrickmehaffy** [comment]
I'm not sure I understand quite what you are asking, can you please clarify a bit?

**Reply by VirAlex** [reply]
Actually I want to use strapi as a CMS with graphql etc ... to work with editable data but I want to know if it is possible to work in front.
For example, I am a user and I filled a form, I send my data like my name etc ... and I click send and it is stored via strapi in a database. With a publishing method for example as can be done with other services like mongoDB or Firebase, ... Thank you

**Reply by derrickmehaffy** [reply]
What you are looking for is called GraphQL Mutations, and yes it is possible. Your GraphQL playground docs should have references to the available mutations
